hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockInputStream.java

```diff
@@ -46,6 +46,7 @@
 import org.apache.hadoop.security.token.Token;
 
 import com.google.common.annotations.VisibleForTesting;
+import org.apache.ratis.thirdparty.io.grpc.Status;
```

Comment:
We need to establish is this is the right import.

Reply:
It is, if you want to catch / process exception from Ratis.